src/CosmosDB/CosmosDB/CosmosDBAccount/NewAzCosmosDBAccount.cs

```diff
@@ -54,13 +57,9 @@ public class NewAzCosmosDBAccount : NewOrUpdateAzCosmosDBAccount
         [PSArgumentCompleter(SDKModel.ServerVersion.ThreeFullStopTwo, SDKModel.ServerVersion.ThreeFullStopSix)]
         public string ServerVersion { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = Constants.LocationHelpMessage)]
-        [ValidateNotNullOrEmpty]
-        public string[] Location { get; set; }
```

Comment:
revert: this was a bug. Update cmdlet should not have Locations as an parameter. Updating Location means Adding region, which is done in a separate cmdlet. Be sure to regenerate the help files.